llvm_mode/pass/AngoraPass.cc

```diff
@@ -524,9 +524,17 @@ void AngoraLLVMPass::resetIndirectCallContext(IRBuilder<> *IRB) {
 }
 
 void AngoraLLVMPass::processCall(Instruction *Inst) {
-  
-  CallInst *CI = dyn_cast<CallInst>(Inst);
-  Function* fp = CI->getCalledFunction();
+
+  Function* fp;
```

Comment:
Here I am being picky, I know. Initialize this to `nullptr`, just to be on the safe side. Someone could add a new branch to the `if-elif-else` where `fp` is not initialized.